### ISSUE TITLE
option (or default?) to render points as zero-length lines

### ISSUE THREAD

**Issue by RossPaterson** [fitnr/svgis#10]
Currently it is possible to control the display of lines and areas using the stylesheet, but point objects are always rendered as a `circle` with radius 1. If a point could be rendered as a zero-length line, i.e. a `line` with the point as start and end point, users would be able to style the point size using the `stroke-width` property.

**Comment by fitnr** [comment]
The `<circle>` element can also be [styled with the `stroke-width` property](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/circle). Are there other reasons for this feature?

**Comment by RossPaterson** [comment]
True, if a bit clunky: one could get a circle of radius n by setting `stroke-width` to 2(n-1). A `line` element can also be styled with a `marker-start` (or `marker-end`) property, to place an arbitrary marker on the point. Of course anyone doing this would have to add a `marker` definition to the SVG file. This could be done with a circle too, I suppose, by giving the circle no fill or stroke and moving the marker by one unit.

**Comment by fitnr** [comment]
With [`calc`](https://developer.mozilla.org/en-US/docs/web/css/calc), doing multiplication is relatively low-effort. 
A more general change might be to employ a `<symbol>` and then a `<use>` element for each point.

**Comment by RossPaterson** [comment]
After a bit of testing: styling a large `stroke-width` on a small `circle` works with Chrome but not with Firefox, librsvg or Inkscape. Styling a `marker-start` on a `circle` only works with Inkscape.

The `<use>` approach seems less flexible: with a stylesheet, one can use different radii or markers depending on values of attributes of the original objects (e.g. types of settlements).
